CRM/Utils/Array.php

```diff
@@ -61,20 +61,17 @@ public static function value($key, $list, $default = NULL) {
    * @return mixed
    *   The value of the key, or null if the key is not found.
    */
-  public static function retrieveValueRecursive(&$params, $key) {
-    if (!is_array($params)) {
-      return NULL;
-    }
-    elseif ($value = CRM_Utils_Array::value($key, $params)) {
-      return $value;
+  public static function retrieveValueRecursive(array $params, string $key) {
+    if (isset($params[$key])) {
```

Comment:
@demeritcowboy so we need to change this to if (!empty() ) to get no change

Reply:
A quick run says yes.

Reply:
OK - I'll put that up

Reply:
#19667